src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/TestStringParser.cs

```diff
@@ -97,7 +96,7 @@ public static Pattern Parse(string testString)
         /// <param name="context">The ParserContext</param>
         /// <param name="results">The List of results which the new condition parameter will be added.</param>
         /// <returns>true </returns>
-        private static void ParseConditionParameter(ParserContext context, List<PatternSegment> results)
+        private void ParseConditionParameter(ParserContext context, List<PatternSegment> results)
```

Comment:
I wrote them in when I had no idea what I was doing lol. I can delete them for you ;)